lib/acme_server/db.ex

```diff
@@ -1,24 +1,36 @@
 defmodule AcmeServer.Db do
   use GenServer
 
+  @type site :: String.t()
+  @type dns :: %{String.t() => String.t()}
+  @type config :: %{site: site, site_uri: URI.t(), dns: dns}
+
   def start_link(config), do: GenServer.start_link(__MODULE__, config)
 
+  @spec store(config, tuple(), map()) :: true
```

Comment:
Here, the types are overly specific. This module doesn't assume anything about keys/values, so we should just use `any()` instead.